app/components/tech-icons/GrammyIcon.tsx

```
@@ -1,23 +1,20 @@
   return (
     <>
       <svg
         className={`hidden sm:block ${className}`}
         width={47}
         height={47}
-        viewBox="0 0 24 24"
+        viewBox="0 0 320 320"
         fill="none"
         xmlns="http://www.w3.org/2000/svg"
         {...props}
       >
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)